src/components/RequestGroupComposition/Window.vue

```diff
@@ -193,7 +193,12 @@ export default {
   },
   computed: {
     topLevelErrors: function() {
-      return extractTopLevelErrors(this.errors);
+      let combinedError = extractTopLevelErrors(this.errors);
```

Comment:
`this.errors` comes from a prop passed down from the parent, and will have errors on specific fields. The `airmassData` can also have an error (if there is an error with visibility), in which case this code below should include that in the error list shown at the top of this component.